### ISSUE TITLE
Improving our documentation

### ISSUE THREAD

**Issue by BenJam** [Open-Source-Collective/osc#2]
Open Source Collective (OSC) now has its own documentation https://docs.oscollective.org/ 🙌  

Sadly the documentation is a little long in the tooth and it desperately needs an overhaul.But our good friends over at OCF have laid the groundwork for us with some excellent work: https://docs.opencollective.foundation/

I'm going to use this issue as the main tracking issue for improvements to the documentation. I'll list some requirements, sections which we can brand away and work on from here. The documentation should:

- [x] explain a little about who we are
     - [x] our mission, values, and focus areas
     - [x] our corporate status, structure, and leadership
     - [x] our team
- [x] explain how our fiscal hosting processes works, and our relationship with Open Collective Inc.
- [x] explain what Open Source Collective does
     - [x] what we offer projects
         - [x] Fiscal sposnoship (inc. model C descriptions)
         - [x] Trademark protection
         - [x] [Employment arrangements](https://app.gitbook.com/o/-LWSZizNMEjL8_DrMNdF/s/-MN_ynR0BjrarSoA9Z-A/~/changes/4WvwgYzOC8gCJdgtuqGu/what-we-offer/employment-payroll-and-benefits)
         - [x] virtual cards
     - [x] what we offer contributors
         - [x] way to support projects (single, recurring etc)
         - [x] corporate options (invoicing, funds, prepaid, gift cards etc)
     - [x] Our fees
- [ ] provide guidance for new Collectives who are getting started
     - [x] who to apply
     - [x] eligibility criteria
     - [x] terms of sponsorship
     - [x] Basics (receiving funds, submitting expenses, payouts)
     - [x] Setting up GitHub Sponsors
     - [ ] Policies
     - [ ] Recommended practices (setup guide)
- [ ] guidance for contributors getting started
     - [x] contributing as an individual
     - [x] corporate contributions (procurement, invoices, legal terms)
     - [x] funds
     - [ ] prepaid balances
     - [ ] gift cards
- [ ] provide routes to both peer support and support from OSC

**Comment by natehn** [comment]
Since this issue was opened, we've made substantial improvements to the docs.

But there is always more to do. Here are a few to-dos that I am moving over from elsewhere:

Branding/messaging:
- [ ] replace the Youtube video on [the front page](https://docs.oscollective.org/) with something more current
- [ ] a comparison page ("How we’re different") that compares OSC to other options for open source projects (other foundations, Tidelift, GitHub sponsors, Patreon, PayPal)
- [x] mission and values page?

Documentation:
- [x] [Google Summer of Code & Season of Docs](https://developers.google.com/season-of-docs/docs/org-payments)
- [x] A bit more about actually getting started
- [ ] integrations: contributor README and contribution embed
- [ ] code of conduct? (or reference to OC guidelines and contributor covenant)
- [x] how to financially contribute (copy OCF page)
     - [x] crypto
     - [x] stock
     - [x] giving via Github Sponsors, company-issued gift cards
     - [ ] other?
- [x] rewrite [Funds page](https://docs.oscollective.org/funding-open-source/funds-for-open-source)?
- [x] expand on the contracts with third parties page
- [ ] potential FAQ questions
     - [ ] "Can I donate stock, cryptocurrency, real estate, a life insurance policy, or another major nontraditional gift?"
     - [ ] "Can we be fiscally sponsored by another group while being hosted by Open Source Collective?"
- [ ] What we offer
     - [x] trademark registration
     - [ ] gift cards - probably short and linking to the OC platform docs
     - [ ] anything else we are missing?

Once program is in place:
- [x] add information about employment and benefits
- [ ] more information about our community work (forums, etc.)
- [x] update acceptance criteria
- [x] more policy documentation? if necessary

**Comment by poohlaga** [comment]
Adding some examples 

- [ ] fiscal host comparison from OCF ([thread here](https://opencollective.slack.com/archives/G46PNRCGP/p1651861673392839))

**Comment by natehn** [comment]
A few more educational content ideas to consider down the road:

- governance structures
- approaches to paying contributors
- contributor experience
- prioritization
- fundraising
- events
- recognition and rewards
- learning from nonprofits on how to manage budgets with irregular grant-cycled funding (and maybe a template for multi-year grant agreements for funders, to set a good example?)

I think it would also be good to try and make sure [these types of questions](https://opencollective.freshdesk.com/a/tickets/46118) are addressed in the docs.

**Comment by BenJam** [comment]
edited some of the information for virtual cards if [you can review ](https://app.gitbook.com/o/-LWSZizNMEjL8_DrMNdF/s/-MN_ynR0BjrarSoA9Z-A/~/diff/~/changes/WaBL3PGA9Cgxvh7hKF6Q/getting-started/expenses) and some [modifications to the welcome/about sections](https://app.gitbook.com/o/-LWSZizNMEjL8_DrMNdF/s/-MN_ynR0BjrarSoA9Z-A/~/diff/~/changes/rUXvftR1CSKYOSVBdkdk/about/what-is-osc) @poohlaga

**Comment by BenJam** [comment]
Amended my old employment arrangements change request that's also [avilaible to review](https://app.gitbook.com/o/-LWSZizNMEjL8_DrMNdF/s/-MN_ynR0BjrarSoA9Z-A/~/changes/4WvwgYzOC8gCJdgtuqGu/what-we-offer/employment-payroll-and-benefits) @poohlaga

**Comment by poohlaga** [comment]
I have gotten requests from collectives to include:

- [x] when to reach out to OSC to review a contract with a vendor

**Comment by BenJam** [comment]
lets review this isues and see what's done/left to be done

**Comment by BenJam** [comment]
As far as I'm concerned, there are a couple of things remaining

- [ ] gift cards
- [ ] integrations (readme)
- [ ] CoC (for community discussions etc)
- [ ] comparison to other kinds of 'foundations'
- [ ] expanded content on giving stock, crypto etc.

**Comment by BenJam** [comment]
closing in favour of asana issue
